cypress/support/commands.js

```diff
@@ -23,3 +23,23 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('selectRandomChild', (parentId) => {
+  cy.get(parentId).click();
+  cy.get('.css-11unzgr').children().then((children) => {
+    const length = children.length;
+    const randomIndex = Math.floor(Math.random() * length);
+    cy.get('.css-11unzgr').children().eq(randomIndex).click();
+  });
+});
+
+Cypress.Commands.add('selectBirthDate', (birthDate) => {
+  cy.get('#dateOfBirthInput').click();
+  cy.get('.react-datepicker__month-select').select(birthDate.month);
+  cy.get('.react-datepicker__year-select').select(birthDate.year);
+
+  const className = 'react-datepicker__day';
+  const day = birthDate.day.padStart(3, '0');
```

Comment:
The `padStart` method is being used here to pad the day with zeros. However, the `padStart` method should use '2' as the first argument instead of '3' to ensure the day is correctly formatted as a two-digit string (e.g., '01', '02', ..., '31').